src/pages-and-resources/discussions/app-config-form/utils.js

```diff
@@ -27,14 +26,34 @@ export const checkStatus = ([startDate, endDate]) => {
   return status;
 };
 
-export const formatDate = (date, time) => (time ? `${date}T${time}` : date);
+export const stringifyDateTime = (date, time) => {
+  if (date && time && time !== 'Invalid date') {
```

Comment:
why `'Invalid date'` check is added here? What's the use case...
todo - discuss that in person.